tests/Stellar/TransactionTests.cpp

```diff
@@ -19,13 +19,12 @@ using namespace std;
 using namespace TW;
 using namespace TW::Stellar;
 
-TEST(StellarTransaction, sign) {
+TEST(StellarTransaction, signFromPhrase) {
```

Comment:
I think we can add a stellar derivation test to `TWHDWalletTests.cpp` and only keep the private key test here